packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2137,6 +2137,25 @@ class _ElementLocationStatsTracker {
       }
       json['newLocations'] = locationsJson;
     }
+
+    // Add in a data structure for the location names.
+    if (newLocations.isNotEmpty) {
+      final Map<String, Map<int, String>> namesJson = <String, Map<int, String>>{};
+      for (final _LocationCount entry in newLocations) {
+        final _Location location = entry.location;
+        if (location.file != null) {
+          final Map<int, String> jsonForFile = namesJson.putIfAbsent(
+            location.file!,
+                () => <int, String>{},
+          );
+          if (location.name != null) {
```

Comment:
nit:
add 
`final name = location.name;`
to eliminate the `!` on the following line.

Reply:
done